docs/administrator-documentation/moderne-platform/how-to-guides/dev-center.md

```diff
@@ -16,11 +19,13 @@ In order to configure any DevCenters, there are two things you need to do (which
 1. You must have either [configured an Organizations service](./organizations-service.md) or [directed the agent to a devCenter.json file](./agent-configuration/configure-agent-files-service.md#devcenterjson-optional).
 2. You must ensure that the [Moderne agent Maven configuration](./agent-configuration/configure-an-agent-with-maven-repository-access.md) only has **one** entry where the recipe source is set to `true`. (Note: this does not apply to one Maven repository configured identically in multiple agents. Only that you cannot have two distinct Maven repositories configured where recipe source is set to `true`.)
 
-### Organizations service
+### Organizations configuration
+
+If you are configuring an Organizations  for the first time, we **strongly** recommend that you use our [file-based approach](agent-configuration#file-based-organizational-configuration) and modify it to meet your needs. By doing so, you will only need to update some CSV files rather than writing your own code.
 
-If you are configuring an Organizations service for the first time, we **strongly** recommend that you use our [Organizations service template](https://github.com/moderneinc/moderne-organizations) and modify it to meet your needs. By doing so, you will only need to update some JSON files rather than writing your own code.
+If you are choosing to not use a file base approach and setup an Organizations service, we **strongly** recommend that you use our [Organizations service template](https://github.com/moderneinc/moderne-organizations) and modify it to meet your needs. By doing so, you will only need to update some JSON files rather than writing your own code.
```

Comment:
```suggestion
If you are choosing to not use a file-based approach on the agent and setup an Organizations service, we **strongly** recommend that you use our [Organizations service template](https://github.com/moderneinc/moderne-organizations) and modify it to meet your needs. By doing so, you will only need to update some JSON files rather than writing your own code.
```